### ISSUE TITLE
The "brightness" game setting has no visible effect in the shader.

### ISSUE THREAD

**Issue by EricDelvalet** [McbeEringi/esbe-2g#8]
Tested with shader version 3.2 on Windows 10. And I have to say first your shader is beautiful 👍 
My PC screen is quite bright and in Vanilla minecraft, I can see enough to go around in a completely dark cave. At the same time, daylight is looking ok.
You shader look extremely contrasted on this screen: I have to lower the screen brightness for daylight but then, night is very very dark. I understand that it is an aesthetic choice but I tried to adjust the game experience to my hardware. I made some research and found:
* I cannot adjust the gamma on the screen
* there is a gfx_gamma parameter hidden in the configuration file but it has no effect in game since ... years?
* there is a brightness adjustment in video setting, but is look like it has **no effect** when your shader is "ON" (that is my bug report :-) ) ( I see no difference at least)
Using this parameter just to change the "overall" brightness will not be very useful (screen adjustment can do that)
But It would be very nice if your shader can take this brightness parameter to adjust the "maximum" daylight brightness, or something like the maximum brightness range ( a bit like a gamma adjustment) . Then we could use the screen adjustment for the darkest game scenes, and "brightness" for the brightest scenes.

(Note: I am programmer and I know nothing about shader but I am ready to experiment with it: I have been looking at your code to see if I can change it myself. I found code that use the "Fancy graphic" switch for example, but I have no idea how to get the "brightness setting" value. I even checked your twitter but found nothing. If you do not want this change in your code base, I can make a clone a test with it) 
Thank for the great work anyway 👍

**Comment by sylcai** [comment]
I would like to agree with this one.
The days are just a tad to bright, where even obsidian gets a very white haze over it, and becomes a dull gray purple. The nights are way to dark when adjusted for the day brightness.
Some sort of gamma interface, or an alternative tonemap, which brings all values a bit closer together, (less high brightness, less intense darkness) would be much appriciated.

**Comment by EzDingoz** [comment]
Agree. Is there anyway to adjust ambient brightness in 2G? On my decent calibrated monitor daylight is way too bright to use this shader at all. (Side note in your 3G version adjusting the data values for rgb day/dusk/night/indoors works to solve this issue and using those 3G terrain shaders with adjustments combined with the 2G rest of the pack then it looks great!).
